src/kernel/src/main.rs

```diff
@@ -33,6 +32,8 @@ use sysinfo::{MemoryRefreshKind, System};
 use thiserror::Error;
 use tty::TtyInitError;
 
+pub use self::arnd::rand_bytes;
+
```

Comment:
I don't think we should re-export any sub-module items at the top-level application.